lib/hubspot/discovery/crm/companies/api/associations_api.rb

```diff
@@ -0,0 +1,21 @@
+require_rel '../../../base_api_client'
+
+module Hubspot
+  module Discovery
+    module Crm
+      module Companies
+        class AssociationsApi
+          def self.api_methods
+            %i[
+              archive
```

Comment:
I know that in other SDKs we don't hardcode final methods, is it possible to do smth similar in ruby?